tests/src/rules/no-cycle.js

```diff
@@ -5,7 +5,7 @@ import { RuleTester } from 'eslint'
 const ruleTester = new RuleTester()
     , rule = require('rules/no-cycle')
 
-const error = message => ({ ruleId: 'no-cycle', message })
```

Comment:
should this then be `test: 'no-cycle'`?

Reply:
What you mean?

Reply:
oh sorry, `type`. You changed a bunch of `ruleID: foo` to be `type: foo` - should they all be changed, rather than just deleting them?

Reply:
I changed some `ruleId: ...` to `type: ...` because these types are already correct, like `ruleId: "ExportNameSpecifier"` to `type: "ExportNameSpecifier"`. In other places they are totally wrong , like `ruleId: "no-cycle"`. Filling out the correct type for these cases is too much work for me.

Reply:
Can these additions made in another PR?

Reply:
Sure, that's fine.